docs/requirements.txt

```diff
@@ -13,5 +13,8 @@ pyyaml>=5.1.0
 requests>=2.20.1
 sqlalchemy>=1.3.0
 tabulate>=0.8.0, <=0.8.3
+fastapi==0.54.1
+uvicorn==0.11.3
+pydantic==1.5.1
```

Comment:
@Hedingber why are those also installed in the client/SDK? shouldn't it be only in the API service container?